src/main/java/training/subsystems/Arm/Elbow/NeoElbow/NeoElbowConstants.java

```diff
@@ -0,0 +1,31 @@
+package training.subsystems.Arm.Elbow.NeoElbow;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+
+public class NeoElbowConstants {
+
+	protected static final int MOTOR_ID = 0;
+
+	protected static final double TOLERANCE = 5;
+
+
+	protected static final double KS = 0;
+
+	protected static final double KG = 0;
+
+	protected static final double KV = 0;
+
+	protected static final double KA = 0;
+
+	protected static final ArmFeedforward ARM_FEEDFORWARD = new ArmFeedforward(KS, KG, KV, KA);
```

Comment:
```suggestion
	protected static final ArmFeedforward FEEDFORWARD = new ArmFeedforward(KS, KG, KV, KA);
```
why the arm, but save the constants and make it a member in the neo class

Reply:
wdym make the constants a member in neo

Reply:
member like CanSparkMax